core/executor/src/wasm_executor.rs

```diff
@@ -461,6 +463,15 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 		this.memory.set(out, &result).map_err(|_| UserError("Invalid attempt to set result in ext_blake2_256"))?;
 		Ok(())
 	},
+	ext_keccak_256(data: *const u8, len: u32, out: *mut u8) => {
+		let result = if len == 0 {
```

Comment:
For documentation purposes, consider specifying the type `[u8; 32]` explicitly.

Reply:
Not sure what you mean here..

Reply:
Basically this

```suggestion
		let result: [u8; 32] = if len == 0 {
```

it allows the reader of the code realize that only `out[0..32]` bytes are written.

Reply:
Well.. that was just copy-pasted code, so to keep it consistent with the other hash impls I'd need to touch more than this PR is meant to...